beacon/endpoints/rest/schemas/alternative.py

```diff
@@ -62,7 +64,7 @@ def ga4gh_phenopackets_biosamples_v10(row):
                 },
                 'body_site': None,
             },
-            'hts_files': None,
+            'hts_files': [jsonb(v) for v in row['files'] or []],
```

Comment:
I think I understand: Your `or []` handles the case when `raw['files']` is `None`.
Well, you should, instead of using `or []`, use coalesce in the database function so that you _always_ return an array. In a strongly-typed language, you wouldn't be able to do that, and that's a good thing. Let's try to mimic that.

Reply:
Exactly, the value can be `None`. I'll change the Postgres function to return `[]` (I'm already doing so in other columns).